xrpl/asyncio/transaction/main.py

```diff
@@ -142,6 +147,72 @@ async def submit_transaction(
     raise XRPLRequestFailureException(response.result)
 
 
+async def safe_sign_transaction_with_multisign(
```

Comment:
This file is already massive - let's move these methods to a separate file in this folder (perhaps called `multisign`).